tests/test_kafka.py

```diff
@@ -140,7 +151,7 @@ def test_pipeline(kerberos_flag, stop_count, jar_uri, keytab_secret, jaas_uri=No
         "--conf", "spark.mesos.driver.secret.filenames=kafka-client.keytab",
         "--conf", "spark.mesos.executor.secret.names={}".format(keytab_secret),
         "--conf", "spark.mesos.executor.secret.filenames=kafka-client.keytab",
-        "--conf", "spark.mesos.task.labels=DCOS_SPACE:/spark",
+        "--conf", "spark.mesos.task.labels=DCOS_SPACE:{}".format(utils.SPARK_APP_NAME),
```

Comment:
This is done [in the CLI](https://github.com/mesosphere/spark-build/blob/master/cli/dcos-spark/submit_builder.go#L651) and iff you want to specify a different DCOS_SPACE, it should be done as a [DC/OS arg](https://github.com/mesosphere/spark-build/blob/master/cli/dcos-spark/main.go#L25) - I can make the PR to change this, however.

Reply:
OK, I'll let you change it in a separate PR.